packages/fxa-payments-server/src/components/Header/index.stories.tsx

```diff
@@ -12,6 +17,7 @@ const userProfile = {
   locale: 'en-US',
   twoFactorAuthentication: false,
   uid: 'UIDSTRINGHERE',
+  metricsEnabled: true,
```

Comment:
All of these stories were failing in CI without this added. I can't help but think there should be a single Profile mock that's pulled in everywhere and modified as needed, I'm going to file an issue for that and a couple of others I came across while spelunking around in payments